hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1198,6 +1200,8 @@ private void finishActiveMasterInitialization(MonitoredTask status)
     }
     // Set master as 'initialized'.
     setInitialized(true);
+    startupTaskGroup.markComplete("Initialization successful");
+    MonitoredTask afterInitialized = startupTaskGroup.addTask("Progress after master initialized");
```

Comment:
So why the previous operations are all tasks, but starting from here, we use the same task?

Reply:
Oh, we can avoid to add this task after the initialize complete, since it brings some confusion. Thank, @Apache9 . Let me change it.